ephios/plugins/eventautoqualification/signals.py

```diff
@@ -0,0 +1,22 @@
+from django.dispatch import receiver
+
+from ephios.core.signals import event_forms, participation_finished
+from ephios.plugins.eventautoqualification.consequences import create_qualification_consequence
+from ephios.plugins.eventautoqualification.forms import EventAutoQualificationForm
+
+
+@receiver(
+    event_forms,
+    dispatch_uid="ephios.plugins.eventautoqualification.signals.configuration_form",
+)
+def configuration_form(sender, event, request, **kwargs):
+    if not request.user.has_perm("core.change_userprofile"):
```

Comment:
maybe it would be nice to display a hint like "this event will grant xy, but you don't have the permission to manage qualification grants"?
but that's nice to have, you can also move that to an issue

Reply:
I built that.